packages/documentation/pages/usage/components/form-fields.vue

```diff
@@ -111,6 +111,17 @@
 				/>
 				<KtFormControllerObject formKey="additionalProps" v-if="componentDefinition.additionalProps.length > 0">
 					<h4>Additional Props</h4>
+					<KtFieldSingleSelect
+						v-if="componentDefinition.additionalProps.includes('actions')"
+						formKey="actions"
+						isOptional
+						label="action"
+						:options="[
+							{ label: 'no action', value: '' },
+							{ label: 'create', value: 'create' },
```

Comment:
This is basically a `KtFieldSingleSelect` that’s used as a boolean. Perhaps `KtFieldToggleGroup` or `KtFieldToggle` is preferable?